### ISSUE TITLE
brew audit error

### ISSUE THREAD

**Issue by fxcoudert** [Homebrew/brew#3907]
`brew audit proj` currently fails with the following backtrace:

```
$ brew audit proj
Error: no block given
Please report this bug:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/utils.rb:112:in `tap'
/usr/local/Homebrew/Library/Homebrew/utils.rb:112:in `odeprecated'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2351:in `fails_with'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/proj.rb:70:in `<class:Proj>'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/proj.rb:1:in `load_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:27:in `module_eval'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:27:in `load_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:269:in `klass'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:89:in `get_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:325:in `from_contents'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:40:in `block in formula_at_revision'
/usr/local/Homebrew/Library/Homebrew/utils.rb:415:in `nostdout'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:40:in `formula_at_revision'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:100:in `block in version_attributes_map'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:26:in `block (2 levels) in rev_list'
/usr/local/Homebrew/Library/Homebrew/utils/popen.rb:14:in `block in popen'
/usr/local/Homebrew/Library/Homebrew/utils/popen.rb:11:in `popen'
/usr/local/Homebrew/Library/Homebrew/utils/popen.rb:11:in `popen'
/usr/local/Homebrew/Library/Homebrew/utils/popen.rb:3:in `popen_read'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:25:in `block in rev_list'
/usr/local/Homebrew/Library/Homebrew/extend/pathname.rb:340:in `block in cd'
/usr/local/Homebrew/Library/Homebrew/extend/pathname.rb:340:in `chdir'
/usr/local/Homebrew/Library/Homebrew/extend/pathname.rb:340:in `cd'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:24:in `rev_list'
/usr/local/Homebrew/Library/Homebrew/formula_versions.rb:99:in `version_attributes_map'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:612:in `audit_revision_and_version_scheme'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:805:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:798:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:798:in `audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:109:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:105:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:105:in `audit'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'
```

---

```
$ brew doctor
Your system is ready to brew.
$ brew config
HOMEBREW_VERSION: 1.5.10-2-g962459a
ORIGIN: https://github.com/Homebrew/brew
HEAD: 962459a3ccc6b97a3ba38a4c64830f3ea5c80ecd
Last commit: 27 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 374e70653a865dc08781dd6244c2a925206bc931
Core tap last commit: 9 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_BINTRAY_KEY: set
HOMEBREW_BINTRAY_USER: fxcoudert
HOMEBREW_DEV_CMD_RUN: 1
HOMEBREW_NO_ANALYTICS_THIS_RUN: 1
HOMEBREW_RUBY_WARNINGS: -W0
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.3.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
Clang: 9.0 build 900
Git: 2.14.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.54.0 => /usr/bin/curl
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
Java: 1.8.0_144
macOS: 10.13.3-x86_64
Xcode: 9.2
CLT: 9.2.0.0.1.1510905681
X11: 2.7.11 => /opt/X11
```

**Comment by fxcoudert** [comment]
I don't understand how we get into `fails_with` given there is no such call in the `proj` formula 🤔

**Comment by maelvalais** [comment]
Hi! @MikeMcQuaid! Do you know when the next version of `brew` (1.5.11) will be released? So that  travis CI can build bottles again 😊 Sorry for bothering!

**Comment by MikeMcQuaid** [comment]
@maelvalais Sorry, you've lost me, why wouldn't Travis CI be able to build bottles?

**Comment by MikeMcQuaid** [comment]
@maelvalais Ah, I read your linked commit. Just released 1.5.11 for your benefit. Thanks for asking so nicely, you are lovely! ❤️
